src/webgl/p5.Shader.js

```diff
@@ -343,6 +343,9 @@ p5.Shader = class {
       this._renderer.uNMatrix.inverseTranspose(this._renderer.uMVMatrix);
       this.setUniform('uNormalMatrix', this._renderer.uNMatrix.mat3);
     }
+    const cameraMat = this._renderer._curCamera.cameraMatrix;
+    const camMat3x3 = cameraMat.createSubMatrix3x3();
+    this.setUniform('uCameraMatrix', camMat3x3.mat3);
```

Comment:
Maybe we can do something similar to what we do for the normal matrix above by:
- putting this block in an `if (this.uniforms.uCameraMatrix) { ... }` to avoid doing extra work for shaders that don't need it
- store a matrix on the renderer for this, similar to `this._renderer.uNMatrix`, to reuse resources